internal/bminventory/inventory.go

```diff
@@ -331,6 +343,52 @@ func (b *bareMetalInventory) getUserSshKey(params installer.GenerateClusterISOPa
 		"groups": [ "sudo" ]}`, sshKey)
 }
 
+func (b *bareMetalInventory) GetDiscoveryIgnition(ctx context.Context, params installer.GetDiscoveryIgnitionParams) middleware.Responder {
```

Comment:
Is there any test coverage for this API?

Reply:
Wow no. I guess I missed that. Good catch

Reply:
:+1: added